db/migrate/20190305150816_drop_elements_from_vm_attrs_hash.rb

```diff
@@ -0,0 +1,83 @@
+class DropElementsFromVmAttrsHash < ActiveRecord::Migration[5.1]
+  def up
+    nics = FakeNic.unscoped.where("attrs like ? or compute_attributes like ?", "%elements:%", "%elements:%")
+    if nics.count > 0
+      say "Correcting serialized nics attributes, total #{nics.count}"
+      nics.find_each(batch_size: 1000) do |compute_attribute|
```

Comment:
No need to explicitly state the batch size everywhere, this is the default size.